rotkehlchen/tests/unit/globaldb/test_globaldb_cache.py

```diff
@@ -238,6 +240,37 @@ def test_velodrome_cache(optimism_inquirer):
     assert mock_notify.call_args_list == [make_call_object(CPT_VELODROME, ChainID.OPTIMISM, 0, 0)]
 
 
+class MockEvmContract:
+    def call(self, *args, **kwargs):
+        if kwargs['arguments'][1] == 0:
+            return [{
+                0: '0x3241738149B24C9164dA14Fa2040159FFC6Dd237',
+                1: '',
+                2: 18,
+                4: 10,
+                7: '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85',
+                10: '0xdFA46478F9e5EA86d57387849598dbFB2e964b02',
+                13: '0xBe3235e341393e43949aAd6F073982F67BFF412f',
+            }]
+        else:
+            return []
+
+
+def test_velodrome_cache_with_no_symbol(optimism_inquirer: OptimismInquirer):
+    """Test a case when a queried pool is not a valid ERC20 token"""
+    def mock_contract(*args, **kwargs):
+        return MockEvmContract()
+
+    with patch('rotkehlchen.chain.evm.contracts.EvmContracts.contract', mock_contract):
+        query_velodrome_like_data(
+            inquirer=optimism_inquirer,
+            cache_type=CacheType.VELODROME_POOL_ADDRESS,
+            msg_aggregator=optimism_inquirer.database.msg_aggregator,
+        )
```

Comment:
this doesn't perform any external query right?

Reply:
yeah, it doesn't make any remote calls